jax/numpy/lax_numpy.py

```diff
@@ -2610,6 +2610,26 @@ def polyval(p, x):
   return y
 
 
+def trim_zeros(a):
+  first = 0
+  for i in a:
+    if i != 0:
+      break
+    first = first + 1
+  return a[first:]
```

Comment:
This should be a private function (`_trim_zeros`). Or, if you're so inclined, you could implement a full jax wrapper for ``np.trim_zeros`` and call that within `polymul`.